airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -381,30 +383,71 @@ def get_num_queued_tasks():
         )
 
 
+seconds_per_unit = {"s": 1, "m": 60, "h": 3600, "d": 86400, "w": 604800}
+sla_interval_pattern = re.compile("[ ]*([0-9]+).*(s|m|h|d|w)[ ]*")
+
+
+def convert_to_seconds(sla_interval):
+    match = re.match(sla_interval_pattern, sla_interval)
+    return int(match.group(1)) * seconds_per_unit[match.group(2)]
+
```

Comment:
Not a big deal, but you could use one of the modules for doing this conversion.  I think there's a few.  Ex:
```
In [9]: from pytimeparse import parse

In [10]: parse('3w')
Out[10]: 1814400
```

Reply:
Sure

Reply:
Tested after switched to `pytimeparse`

<img width="936" alt="Screen Shot 2021-03-22 at 11 49 52" src="https://user-images.githubusercontent.com/624603/112042470-c8cde980-8b04-11eb-9e74-0ff67fe1c710.png">